build-tools/packages/build-cli/docs/release.md

```diff
@@ -6,6 +6,8 @@ Release commands are used to manage the Fluid release process.
 * [`flub release`](#flub-release)
 * [`flub release fromTag TAG`](#flub-release-fromtag-tag)
 * [`flub release history`](#flub-release-history)
+* [`flub release prep PACKAGE_OR_RELEASE_GROUP`](#flub-release-prep-package_or_release_group)
+* [`flub release prepare PACKAGE_OR_RELEASE_GROUP`](#flub-release-prepare-package_or_release_group)
```

Comment:
What is the difference between prep and prepare? By description, looks like both are same.

Reply:
They're the same. It's an alias for the same command. I'll remove it, though, because I didn't expect it to document it
as two separate commands. That's confusing.